### ISSUE TITLE
[Suggestion/Feature Request] A couple suggestions

### ISSUE THREAD

**Issue by Hazmatik** [McJty/RFTools#662]
1. A way to camouflage RFtools blocks, most specifically the Elevator.

2. Allow screens to be placed flat on top/bottom of blocks.

**Comment by McJty** [comment]
1. Doing that in general is not very easy. I could do it for some specific blocks though like the elevator.

2. yes that would be nice but slightly complicated atm. It is on my todo however

**Comment by DRTR0** [comment]
Could just use the same method you use to paint blocks from the shield.

**Comment by McJty** [comment]
Yes but that's possible because those shield blocks are made for that. All other blocks are not and that's not very easy

**Comment by mcdodge34** [comment]
Bringing this suggestion back if its ever becomes possible, that would be amazing for the elevator blocks especially, and perhaps add a door control that would only open the doors if the elevator is at the current floor, I think I can manage to do a system for this right now if the elevator block output a redstone signal when its at a floor, I have to fiddle out a bit with this right now.

**Comment by McJty** [comment]
The elevator does output a redstone signal when it is at a floor. That's what the red arrow is for at the back

**Comment by mcdodge34** [comment]
Good, will have to find a way to get the door system now that I know I can count on a redstone signal. I found a nice sliding door from Malisis doors that should fit nicely. What is the maximum size of the elevator, can I do a 4x4 or its 3x3 limited platform size.

**Comment by uecasm** [comment]
> The elevator does output a redstone signal when it is at a floor. That's what the red arrow is for at the back

Wish I'd known that a few days ago.  I just set up a system using a Sensor to detect the elevator platform being present. :man_facepalming: (Works well, and I can even hide it behind a wall on most floors due to the sensor's 3-block mode.  Though it doesn't work on some floors due to layout, and this would let me fix that.)

I'm using a Shield for the door, although currently in 1.10 this doesn't support using another block texture yet, which would be nice.
